.github/pull_request_template.md

```diff
@@ -1,66 +1,58 @@
-- [ ] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)
-
-#### Other Checks
-
-- [ ] This PR has either a `[type]` label or a `[skip-changelog]` label.
-- [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
-- [ ] This PR adds/removes an experimental interfaces and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
-- [ ] I tagged two reviewers because this PR makes queries to the database or I think it might have some security impact.
-
-### Screenshots
-
-<!-- If your change has a visual component, add a screenshot here. A "before" screenshot would also be helpful. -->
-
-| Before | After |
-| ------ | ----- |
-|        |       |
-
-### Testing
-
-#### Automated Tests
-* [ ] Changes in this PR are covered by Automated Tests.
-  * [ ] Unit tests
-  * [ ] E2E tests
-
-#### User Facing Testing
+## Testing Instructions
 
 <!-- Write these steps from the perspective of a "user" (merchant) familiar with WooCommerce. No need to spell out the steps for common setup scenarios (eg. "Create a product"), but do be specific about the thing being tested. Include screenshots demonstrating expectations where that will be helpful. -->
 
+_Please consider any edge cases this change may have, and also other areas of the product this may impact._
+
 1.
 2.
 3.
 
 * [ ] Do not include in the Testing Notes <!-- Check this box if this PR can't be tested (ie: it makes changes to tests, coding standards, docblocks, etc.). -->
 * [ ] Should be tested by the development team exclusively <!-- Check this box if this PR should be tested by the development team exclusively (ie: it doesn't include user-facing changes or it can't be tested without manually modifying the code). -->
 
-### WooCommerce Visibility
+## Screenshots or screencast
+
+<!-- Any screenshots of UI changes will be helpful to include here. Leave blank if not applicable. -->
+
+| Before | After |
+| ------ | ----- |
+|        |       |
 
-<!-- Check this [this doc](../docs/blocks/feature-flags-and-experimental-interfaces.md) to see if the change is visible in WC core or not (part of the feature plugin or experimental)-->
+## WooCommerce Visibility
+
+<!-- Check this documentation link (../docs/blocks/feature-flags-and-experimental-interfaces.md) to see if the change is visible in WooCommerce core, part of the feature plugin, or experimental. -->
+Required:
 
 * [ ] WooCommerce Core
 * [ ] Feature plugin
 * [ ] Experimental
+* [ ] N/A
+
+## Checklist
 
-### Performance Impact
+Required:
+* [ ] This PR has either a `[type]` label or a `[skip-changelog]` label.
+* [ ] This PR is assigned to a milestone.
```

Comment:
When would this be checked? We don't assign a milestone when opening the PR, but rather when closing it.

Reply:
AFAIU, this checkbox functions as a verification step to ensure that the milestone has been assigned to the pull request (PR) after it has been merged. This precaution is taken due to occasional instances where our [GitHub action job](https://github.com/woocommerce/woocommerce-blocks/blob/e7daa943738016ce95b2e9f3ff6457929fa5306f/.github/workflows/approved-with-labels.yml#L16) doesn't perform as expected. If my understanding is accurate, maybe we should consider rephrasing this checkbox for clarity. 🤔